### DISCUSSION TITLE
To extract items from RSS feed and preserve them in archive_org

### DISCUSSION THREAD

**Discussion by squromiv** [ArchiveBox/ArchiveBox#1134] [Q&A]
I am newbie to ArciveBox. I need to save articles from RSS feed into archive.org.
Test feed:
https://demo.freshrss.org/i/?a=rss&rid=643047f465975&hours=12

So I have launched ArchiveBox with Docker compose. Set options to apply.
<details>
<summary>screen1</summary>
<img src="https://user-images.githubusercontent.com/15985877/230714529-16b7ba10-5bd1-4588-a75d-81691f7f76e7.png"/>
</details>

Result
<details>
<summary>screen2</summary>
<img src="https://user-images.githubusercontent.com/15985877/230714408-4e165af4-3f4e-49fc-84e2-f9591fc1dc3a.png"/>
</details>

Am I doing anything wrong?

**Comment by pirate** [comment]
Set depth=1 and it should work fine

**Reply by squromiv** [reply]
> Set depth=1 and it should work fine

Zero URLs parsed 
![arch3](https://user-images.githubusercontent.com/15985877/230757143-13df3803-cceb-4c65-9672-57ccaca53852.png)

**Reply by pirate** [reply]
Try deleting the snapshot it added for the original URL,. then re-adding it with depth=1. I think it's not crawling it again because it's seeing the old snapshot.

**Comment by squromiv** [comment]
> Try deleting the snapshot it added

There is no snapshots right now, but result is the same.
![arch4](https://user-images.githubusercontent.com/15985877/230849562-bfdee64f-7b34-41d7-b83b-2bd253d1e2d6.png)

**Reply by pirate** [reply]
Hmm ok, can you try parsing it as plain text instead of RSS. I want to see if it's having an issue on the pulling side or the parsing side.

If it works with txt but is missing the RSS metadata then we know it's a bug in the RSS parser.

**Comment by squromiv** [comment]
Have checked. With "Generic TXT" and "depth=1" options it extracts plenty of links. 
![arch6](https://user-images.githubusercontent.com/15985877/230973953-d3d5d510-1996-4470-a0ec-1c65a3540a2e.png)

**Reply by pirate** [reply]
ok, makes sense, maybe theres something incompatible between the RSS format in that file and our RSS parser. I cant promise a rapid fix so I'd suggest using the txt import for now. I'll pull in the titles automatically as it archives but it wont have the correct bookmarked date or any tags from the RSS file.
